src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+
+        }
+
+        for (int a = 3; a < 6; a++) {
+            figures[a] = figureSupplier.getDefaultFigure();
+        }

```

Comment:
```suggestion
        for (int i = 0; i < 3; i++) {
            figures[i] = figureSupplier.getRandomFigure();

        }

        for (int a = 3; a < 6; a++) {
            figures[a] = figureSupplier.getDefaultFigure();
        }

```

use one loop for this